src/components/RepCard/RepCard.js

```diff
@@ -0,0 +1,23 @@
+import "./RepCard.css";
+
+function RepCard(props) {
+  const repCardClasses = `rep-card ${props.party}`;
+  return (
+    <div className={repCardClasses}>
```

Comment:
You could just do ``<div className={`rep-card ${props.party}`}`` so that you don't need to define repCardClasses